src/fides/api/api/v1/endpoints/pre_approval_webhook_endpoints.py

```diff
@@ -0,0 +1,258 @@
+from typing import List
+
+from fastapi import Body, Depends, Security
+from fastapi_pagination import Page, Params
+from fastapi_pagination.bases import AbstractPage
+from fastapi_pagination.ext.sqlalchemy import paginate
+from fideslang.validation import FidesKey
+from loguru import logger
+from pydantic import conlist
+from sqlalchemy.orm import Session
+from starlette.exceptions import HTTPException
+from starlette.status import HTTP_200_OK, HTTP_400_BAD_REQUEST, HTTP_404_NOT_FOUND
+
+from fides.api.api import deps
+from fides.api.common_exceptions import KeyOrNameAlreadyExists
+from fides.api.db.base_class import get_key_from_data
+from fides.api.models.policy import (
+    PolicyPreWebhook,
+)
+from fides.api.models.pre_approval_webhook import PreApprovalWebhook
+from fides.api.oauth.utils import verify_oauth_client
+from fides.api.schemas import pre_approval_webhooks as schemas
+from fides.api.util.api_router import APIRouter
+from fides.api.util.connection_util import get_connection_config_or_error
+from fides.common.api import scope_registry as scopes
+from fides.common.api.v1 import urn_registry as urls
+
+router = APIRouter(tags=["Pre Approval Webhooks"], prefix=urls.V1_URL_PREFIX)
```

Comment:
This router needs to be imported into src/fides/api/api/v1/api.py to actually register the endpoint!